assets/js/util/test/readableLargeNumber.js

```diff
@@ -58,8 +58,120 @@ const valuesToTest = [
 	],
 ];
 
+const numbersBelowOneThousand = [
+	[
+		1,
+		'1',
+	],
+	[
+		450,
+		'450',
+	],
+	[
+		550,
+		'550',
+	],
+	[
+		999,
+		'999',
+	],
+];
+
+const numbersBetweet1000And10000 = [
+	[
+		1000,
+		'1K',
+	],
+	[
```

Comment:
Might be good to add a test for a number that would round down by default—just for completeness.

```suggestion
	[
		1050,
		'1K',
	],
	[
```